src/cmds/qsub_sup.c

```diff
@@ -264,16 +265,19 @@ get_script(FILE *file, char *script, char *prefix)
 	while ((in = pbs_fgets(&s_in, &s_len, file)) != NULL) {
 		if (!exec && ((sopt = pbs_ispbsdir(s_in, prefix)) != NULL)) {
 			/* Check if this is a directive line that should be extended */
-			extend_len = pbs_extendable_line(in);
-			if (extend_len >= 0) {
-				in[extend_len] = '\0'; /* remove the backslash (\) */
+			extend_loc = pbs_extendable_line(in);
+			if (extend_loc >= 0) {
+				in[extend_loc] = '\0'; /* remove the backslash (\) */
+				extend_in = NULL;
+				extend_len = 0;
```

Comment:
You want to set extend_in to NULL and extend_len to 0 at the top of the function once.  After the first call to pbs_fgets_extend() there will be memory allocated to extend_len, so it can be reused.